site/config.toml

```diff
@@ -0,0 +1,27 @@
+baseURL = "/"
+languageCode = "en-us"
+title = "Google Cloud SDK Javadocs"
+
+[module]
+proxy = "direct"
+[[module.imports]]
+path = "github.com/google/docsy"
+
+[[module.imports]]
+path = "github.com/google/docsy/dependencies"
+contentDir = "content/"
+pygmentsCodeFences = true
+pygmentsUseClasses = false
+pygmentsUseClassic = false
+pygmentsStyle = "tango"
+enableInlineShortcodes = true
+
+[params]
+libraries_bom_version = "libraries-bom: v26.17.0"
```

Comment:
Is this correct, or is the version parsing incorrectly adding the label?

Reply:
This is correct. This variable is being used for the actual site rendering here:

![image](https://github.com/googleapis/java-cloud-bom/assets/65933803/a0b471ff-1948-4869-a97c-1f4c1db37918)